cypress/downloads/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
Isnt it supposed to be `.*` instead of `*`

Reply:
I tested it by putting some arbitrary file in the downloads directory. After updating `.gitignore` as done here, the file was no longer shown with `git status`. Do you want me to change to `.*` and test that?

Reply:
Yes, can you test it?

Reply:
`.*` does not do it. Only `*`.

Reply:
Okay